nodePackages/default.nix

```diff
@@ -2791,6 +2801,9 @@
     newrelic_1-25-0 = callPackage ./newrelic/1.25.0.nix {};
     next-tick = callPackage ./next-tick/0.2.2.nix {};
     next-tick_0-2-2 = callPackage ./next-tick/0.2.2.nix {};
+    ng-redux = callPackage ./ng-redux/3.4.0-beta.1.nix {};
+    ng-redux_quill3 = callPackage ./ng-redux/quill3.nix {};
```

Comment:
If this is identical to `./ng-redux/3.4.0-beta.1.nix`, just call that package and name it `ng-redux-quill3`. So:
```
ng-redux_quill3 = callPackage ./ng-redux/3.4.0-beta.1.nix {};
```

Same with redux below, although at this point I'm not sure whether it's actually necessary, since I don't see any conflicts (every one is the latest version, right?). What is the conflict you're expecting in the future?